web/src/views/DandisetLandingView/ListingComponent.vue

```diff
@@ -80,24 +30,106 @@ export default {
       type: [Object, Number, String, Array],
       required: true,
     },
-    root: {
-      type: Boolean,
-      default: false,
+    field: {
+      // The key in the parent object
+      type: String,
+      required: true,
     },
   },
   computed: {
+    renderComponent() {
```

Comment:
```suggestion
    renderedComponent() {
```
`renderComponent` sounds like an action rather than a string.